modules/changelog.sh

```diff
@@ -1,4 +1,9 @@
 function changelog {
+if [ -d "tests" ]; then
+  cp "/usr/lib/node_modules/@treehouses/cli/CHANGELOG.md" .
+else
+  cp "/usr/lib/node_modules/@treehouses/cli/CHANGELOG.md" ../.
+fi
```

Comment:
I may be mistaken but I think @dogi left checking the directory for `tests/changelog.bats`
Also you didn't remove the CHANGELOG file....

Reply:
Fixed - it now removes the duplicate CHANGELOG file.

Reply:
> ![jamiecomparechangelog](https://user-images.githubusercontent.com/23529006/114253376-efe74080-9977-11eb-93a1-5dfe31ffb0eb.jpg)
> 
> The changelog compare still shows the changes between new version first and old version last. I don't know if the requirements are strict with only comparing old first and new last. But I'll probably leave out the changes and keep the "Did you mean..." line.
> 
> Anyways it looks a lot better than showing all the changes.

Are you saying you want an error handling message when when comparing changes with the new version first and old version last?

And you're correct, the way the code is written now, the command will 'autocorrect' by displaying the changes as if the versions were switched around, then suggest the correct way to write the command:
![image](https://user-images.githubusercontent.com/61807745/115820083-fda7b780-a3b4-11eb-9524-b8cdb2e6eea5.png)